### PR TITLE
Remove protege references

### PR DESCRIPTION
**Pull request by schlotze**
Removes the references to Protege libraries due to IP review conflict

### CODE REVIEW

**Review by philipreimer — CHANGES_REQUESTED** [review]

**Comment by phkrief** [comment]
Just to be sure: Did you manage to integrate these changes?
Thx

**Comment by schlotze** [comment]
we are still working on it

File: context-core/src/main/java/de/atb/context/rdf/manager/ServiceRegistryRepository.java [hunk 1]
```diff
@@ -57,9 +52,7 @@
             .getProperty("user.home")
             + File.separator
             + ".context";
```
[the remark this answers is not in the document]
Comment (schlotze) [reply context-core/src/main/java/de/atb/context/rdf/manager/ServiceRegistryRepository.java:54]: done

File: context-core/src/main/java/de/atb/context/rdf/registry/Vocabulary.java [hunk 2]
```diff
@@ -26,7 +26,6 @@
  * Vocabulary class to provide access to the Manchester OWL API representatives for
  * various entities in the ontology used to generate this code.
  *
- * Generated by Protege (http://protege.stanford.edu).<br>
  * Source Class: ${javaClass}
```
[the remark this answers is not in the document]
Comment (schlotze) [reply context-core/src/main/java/de/atb/context/rdf/registry/Vocabulary.java]: class is deleted due to further clean up

File: context-core/src/main/java/de/atb/context/rdf/manager/ServiceRegistryRepository.java [hunk 3]
```diff
@@ -118,175 +109,28 @@ public void initializeRepository() {
...
-                            .addHasDeployerSWServices(serviceWrapper);
-                }
-                factory.saveOwlOntology();
-                return true;
-            } catch (OWLOntologyStorageException ex) {
-                logger.error(ex.getMessage());
-            }
-        }
-        return false;
+        return initialized && (deployer != null && deployer.getConfig() != null
```
[the remark this answers is not in the document]
Comment (schlotze) [reply context-core/src/main/java/de/atb/context/rdf/manager/ServiceRegistryRepository.java]: class is deleted due to further clean up

File: context-core/src/main/java/de/atb/context/rdf/manager/ServiceRegistryRepository.java [hunk 4]
```diff
@@ -118,175 +109,28 @@ public void initializeRepository() {
...
         if (initialized) {
-            Deployer_OntologyWrapper deployerWrapper = factory
-                    .getDeployerOntologyWrapper(id);
             ConnectedDeployer connectedDeployer = new ConnectedDeployer();
-            SW_Service_Configuration_OntologyWrapper deployerConfigWrapper = factory
-                    .getSWServiceConfigurationOntologyWrapper(id
-                            + CONFIGURATION);
             // TODO: there is a lot of duplicated code `serviceInfo.set...` in this class
             // this should be refactored into a separate method
             ServiceInfo deployerConfig = new ServiceInfo();
```
[the remark this answers is not in the document]
Comment (schlotze) [reply context-core/src/main/java/de/atb/context/rdf/manager/ServiceRegistryRepository.java]: class is deleted due to further clean up

File: context-core/src/main/java/de/atb/context/services/manager/ServiceManager.java [hunk 5]
```diff
@@ -364,10 +358,10 @@ public static synchronized Client getDynamicWebservice(final String wsdl_address
             try {
                 Constructor<?> constructor = serviceClass.getConstructor();
                 T instance = (T) constructor.newInstance();
-                if (instance instanceof ServiceRegistryService) {
-                    ServiceRegistryService server = (ServiceRegistryService) instance;
-                    ServiceManager.server = server.getServerInstance();
-                }
+//                if (instance instanceof ServiceRegistryService) {
```
Comment (philipreimer) [inline context-core/src/main/java/de/atb/context/services/manager/ServiceManager.java:361]: remove commented code

**Review by philipreimer — CHANGES_REQUESTED** [review]

File: context-core/src/main/java/de/atb/context/services/manager/ServiceManager.java (hunk 5 above)
Comment (schlotze) [reply context-core/src/main/java/de/atb/context/services/manager/ServiceManager.java]: [x] done

**Review by philipreimer — APPROVED** [review]

**Pull request merged**